tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -118,6 +118,23 @@ def test_concatenate_list(self):
             == "Milk, Eggs, Flour, Water"
         )
 
+    def test_add(self):
+        assert self.transforms.add(1, 2) == 3
+        assert self.transforms.add("1", 2) == 3
+
+    def test_format_ordinal(self):
+        assert self.transforms.format_ordinal(1, "a_or_an") == "a 1st"
+        assert self.transforms.format_ordinal(2, "a_or_an") == "a 2nd"
```

Comment:
Are we interested in just these ordinals, should we be testing more, and looping, not sure on this one.

Reply:
From what I know, we're only interested in ordinals, hence the transform name. I can add some more assertions but wouldn't looping through a block be a bit excessive?